src/libsync/abstractnetworkjob.h

```diff
@@ -59,11 +59,32 @@ class OWNCLOUDSYNC_EXPORT AbstractNetworkJob : public QObject {
 
     qint64 timeoutMsec() { return _timer.interval(); }
 
+    // These methods are used to replace the direct calls that were used to fetch the same information.
+    // Can and should be overloaded when needed by specific implementation.
+
+    virtual void abortNetworkReply();
+
+    virtual QUrl getUrl();
```

Comment:
The client's coding style is losely based on the Qt coding style. There should not be "get" in front of the function.